kolibri/core/assets/test/heartbeat.spec.js

```diff
@@ -1,3 +1,4 @@
+import mock from 'xhr-mock';
```

Comment:
I'd be interested in using these library here to mock the `Resource` class instead of what I've done with the `JestMockResource` thing. I think it's an improvement because it would actually move the test coverage of the tests using the mocks down a little further down to the DRF layer, esp. if we use more specific URLs). If possible, it might be an improvement to actually use the relevant URLs for the heartbeat API instead of what I think is the a catch-all `/.*`

Reply:
Yeah, I think that would be good overall - we could probably do a minor refactor on your `JestMockResource` class to just use the xhr-mock under the hood.